DEVNOTES.md

```diff
@@ -18,18 +18,21 @@ environment by looking at the deployed configs in https://duos-k8s.dsde-{%ENV%}.
 `dev`. Certain features are available only in specific environments. Setting the `env` value to the desired environment 
 will simulate it for local development. The installation steps outlined in this step can also be completed using the 
 [render-configs.sh](scripts/render-configs.sh) script which can generate all required files for local development.
+
 ```
 cp config/base_config.json public/config.json
 ```
 
 Ensure that your `/etc/hosts` file has an entry for `local.dsde-dev.broadinstitute.org`
+
 ```properties
 127.0.0.1	local.dsde-dev.broadinstitute.org
 ```
 
 Download cert files from dev project (requires access to correct project - see [DUOS team members](https://github.com/orgs/DataBiosphere/teams/duos) for more specifics). Cert files are regenerated on a 3-month rotation so these will need to be updated when they are expired. The following commands need to be done on the Broad VPN.
+
 ```shell
-gcloud container clusters get-credentials --zone us-central1-a --project <project> terra-dev
+gcloud container clusters get-credentials --zone us-central1-a --project broad-dsde-dev terra-dev
```

Comment:
This is the main change, we mention downloading the cert files from the dev project, so this is specified.

Reply:
I thought the correct command was: gcloud container clusters get-credentials terra-dev --zone us-central1-a --project broad-dsde-dev. but maybe these have the same effect?

Reply:
I believe that you can run render-configs and it will pick up on that project by default - I happened to also make a round of changes to the devnotes in[ this PR](https://github.com/DataBiosphere/duos-ui/pull/2745/files#diff-e6fecc1d2ecdd042c611b8f62f78af878293785f983f410a11453b2ff6dcdeafL30), what do you think?

Reply:
I'll close this one.